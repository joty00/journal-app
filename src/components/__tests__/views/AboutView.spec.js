import { shallowMount } from '@vue/test-utils'
import { describe, expect, test } from 'vitest'
import AboutView from '@/views/AboutView.vue'

describe('Pruebas en el About View', () => {

    test('debe renderizar el componente correctamente ', () => {
        const wrapper = shallowMount(AboutView)
        expect(wrapper.html()).toMatchSnapshot()
    })




})