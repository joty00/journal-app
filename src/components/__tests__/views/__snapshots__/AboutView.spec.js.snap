// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Pruebas en el About View > debe renderizar el componente correctamente  1`] = `
"<div class=\\"about\\">
  <h1>This is an about page</h1>
</div>"
`;
